templates/keepalived.service.j2

```diff
@@ -0,0 +1,23 @@
+#jinja2: lstrip_blocks: True
```

Comment:
can you rename this template keepalived-snap.service.j2 ? I think it makes it more obvious it's for snaps only.

Reply:
https://github.com/evrardjp/ansible-keepalived/pull/161/commits/60a5badcdc7c419692aa904e630aa5f44f2bfa0b should be it